cloud/azure/service-bus/detectors-service-bus.tf

```diff
@@ -105,3 +105,36 @@ resource "signalfx_detector" "server_errors" {
     parameterized_body    = local.rule_body
   }
 }
+
+resource "signalfx_detector" "throttled_requests" {
+  name = format("%s %s", local.detector_name_prefix, "Azure Service Bus throttled requests rate")
+
+  program_text = <<-EOF
+    base_filter = filter('resource_type', 'Microsoft.ServiceBus/namespaces') and filter('primary_aggregation_type', 'true') and ${module.filter-tags.filter_custom}
+    A = data('ThrottledRequests', filter=base_filter and ${module.filter-tags.filter_custom})${var.throttled_requests_aggregation_function}${var.throttled_requests_transformation_function}
+    B = data('IncomingRequests', filter=base_filter and ${module.filter-tags.filter_custom})${var.throttled_requests_aggregation_function}${var.throttled_requests_transformation_function}
+    signal = data('(A/B).scale(100).fill(0)', filter=base_filter and ${module.filter-tags.filter_custom})${var.throttled_requests_aggregation_function}${var.throttled_requests_transformation_function}.publish('signal')
```

Comment:
you should not use data function for formula if it comes from generator please tell me it should not.